test/system/components/student/tracks_list_test.rb

```diff
@@ -0,0 +1,90 @@
+require "application_system_test_case"
+
+module Components
+  module Student
+    class TracksListTest < ApplicationSystemTestCase
+      include TableMatchers
+
+      test "shows tracks" do
+        track = create :track, title: "Ruby"
+        user = create :user
+        concept_exercises = Array.new(10).map { create :concept_exercise, track: track }
+        practice_exercises = Array.new(10).map { create :practice_exercise, track: track }
+        create :user_track, track: track, user: user
+        concept_exercises.sample(5).map { |ex| create :concept_solution, user: user, exercise: ex }
+        practice_exercises.sample(5).map { |ex| create :practice_solution, user: user, exercise: ex }
+
+        visit test_components_student_tracks_list_url
+
+        row = {
+          "Icon" => lambda {
+            assert_css "img[src='https://assets.exercism.io/tracks/ruby-hex-white.png'][alt='icon for Ruby track']"
+          },
+          "Title" => "Ruby",
+          "Num concept exercises" => 10,
+          "Num practice exercises" => 10,
```

Comment:
@kntsoriano Try and keep things like number of concept/practice exercises as different values to ensure we're not silently calling the wrong method. Both of these could be calling num_concepts right now and we wouldn't know.

Reply:
Also on this, the tests will be faster the less objects we create, so using `2` and `3` would be a better option than `8` and `10`.